libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/loadConnectingAttributesMatrix.ts

```diff
@@ -82,7 +85,7 @@ export function* loadConnectingAttributesMatrix(
             connectingAttributeMatrix[column] = connectingAttributeMatrix[column]
                 ? connectingAttributeMatrix[column]
                 : [];
-            const connectingAttributes = connectingAttributeRefs[connectingAttributeIndex++].map((ref) =>
+            const connectingAttributes = connectingAttributeRefs[connectingAttributeIndex++]?.map((ref) =>
```

Comment:
This is sketchy, do we want to increment the counter even if the given index is undefined?

Reply:
Yes, there could be items in `connectingAttributeRefs` after the undefined one for which we might need to get data